src/commands/cdxgen/cmd-cdxgen.ts

```diff
@@ -0,0 +1,170 @@
+// import meowOrDie from 'meow'
+import process from 'node:process'
+
+import yargsParse from 'yargs-parser'
+
+import { pluralize } from '@socketsecurity/registry/lib/words'
+
+import { runCycloneDX } from './run-cyclonedx.ts'
+import { getFlagListOutput } from '../../utils/output-formatting.ts'
+
+import type { CliCommandConfig } from '../../utils/meow-with-subcommands.ts'
+
+// TODO: convert yargs to meow. Or convert all the other things to yargs.
+const toLower = (arg: string) => arg.toLowerCase()
```

Comment:
In this case we're just using `yargsParse`  not full `yargs`. This is essentially a polyfill until Node's [`util.parseArgs`](https://nodejs.org/api/util.html#utilparseargsconfig) is more solid.

Reply:
Sure, but we should change that to use meow, right? Be consistent?

Reply:
Yes.